jujugui/static/gui/src/app/components/profile/profile.js

```diff
@@ -97,6 +101,15 @@ class Profile extends React.Component {
         }
       });
     }
+
+    // If viewing a user profile that is not yours then do not show the
```

Comment:
Isn't it better to do the opposite, so that the risk is not showing something that should not be there, but not showing something that should be there, so a false negative which is less risky?